modules/tide_content_collection/src/SearchApiIndexHelper.php

```diff
@@ -301,7 +301,7 @@ public function buildEntityReferenceFieldFilter(IndexInterface $index, string $f
    */
   public function getIndexedNodeField(IndexInterface $index, string $node_field_name) : ?string {
     $index_fields = &drupal_static(__CLASS__ . '::' . __METHOD__);
-    if (array_key_exists($node_field_name, $index_fields)) {
+    if (isset($index_fields[$node_field_name])) {
```

Comment:
@joshua-salsadigital can you please explain this change? Do you know the difference between `array_key_exists()` and `isset()`? Did you see these lines below?
```
    $index_fields[$node_field_name] = NULL;
    return NULL;
```

Reply:
isset() will not consider null values. Which is needed n this case.
When the form is access on the field edit (default value). 
`array_key_exists() expects parameter 2 to be array, null given`

Reply:
@joshua-salsadigital `null values. Which is needed n this case` is not correct. NULL value is required by the follow lines `$index_fields[$node_field_name] = NULL;`. The method specifically sets the value to NULL hence it uses `array_key_exists` instead of `isset`.

To fix the `null given` error, the default value `[]` needs to be passed to `drupal_static`:
```
$index_fields = &drupal_static(__CLASS__ . '::' . __METHOD__, []);
```

Reply:
Implemented $index_fields = &drupal_static(__CLASS__ . '::' . __METHOD__, []);